rero_ils/modules/documents/dojson/contrib/marc21tojson/rero/model.py

```diff
@@ -130,83 +131,90 @@ def marc21_to_contribution(self, key, value):
                 agent['type'] = 'bf:Organisation'
 
     # we do not have a $ref
+    agent_data = {}
```

Comment:
Should we use `entity_data` instead of `agent_data`?

Reply:
Agent makes sense for contribution. It is a special type of entity. We can leave at is I think.